wren-ui/src/apollo/server/repositories/schemaChangeRepository.ts

```diff
@@ -0,0 +1,33 @@
+import { Knex } from 'knex';
+import { BaseRepository, IBasicRepository } from './baseRepository';
+
+export interface SchemaChange {
+  id: number; // ID
+  projectId: number; // Reference to project.id
+  change: string; // Schema change in JSON format
```

Comment:
In PG, the return data type of these two column will be object instead of string.
Either change the column data type defined in the migration file to text or change typescript definitions here to object and do JSON.parse when the return datatype from knex is string.

Reply:
Can refer to the method transformFromDBData in the threadResponseRepository.ts